src/components/PostList.tsx

```diff
@@ -65,9 +79,11 @@ const PostList = ({ feedType }: PostListProps): JSX.Element => {
       ? [feedItem.content.tag]
       : feedItem.content.tag;
 
-    return tagList
+    const formattedTagList = tagList
       .filter((tag) => tag?.name !== "")
       .map((tag) => (tag?.name || "").replace("#", ""));
+
+    return formattedTagList.map((tag) => "#" + tag);
```

Comment:
Let's not hold up the PR for this but replace will also take a regular expression.  So you could replace map.replace.map with just a map.replace:
```script
const testStrs = ["hashtag","#hashtag","##hashtag"]
const regex = /^[\#]*/
console.log(testStrs.map(s => s.replace(regex,"#")))
```
this results in `["#hashtag", "#hashtag", "#hashtag"]`